CHANGELOG.md

```diff
@@ -25,6 +25,7 @@
 
 ## 28.1.0
 
+* Bump `govuk-frontend` from 3.14.0 to 4.0.0 ([PR #2355](https://github.com/alphagov/govuk_publishing_components/pull/2533))
```

Comment:
This changelog entry doesn't match the PR. It should also appear beneath an `Unreleased` heading.